src/app/global/components/app-breadcrumbs/app-breadcrumbs.vue

```diff
@@ -85,12 +88,14 @@
                 </cv-breadcrumb-item>
               </template>
             </cv-breadcrumb>
-            <cv-tag
-              v-if="$route.name === 'river-index' && riverIndexData"
-              kind="blue"
-              :label="`Rivers Found: ${riverIndexData.length}`"
-              :disabled="false"
-            />
+            <a href="#nwi-rivers-table">
+              <cv-tag
+                v-if="$route.name === 'river-index' && riverIndexData"
```

Comment:
Well if we hid breadcrumbs on the index route the counter would need to be pulled out. Initially I added it to the breadcrumbs component because I wanted it to live in the top right of the UI across from the crumbs. Figured just putting it into the same div with flex and justify-content space between was the quickest / cheapest solution.

Reply:
Sorry just to reiterate the first of my two questions here since you're adding `isRiverIndex` to the component anyway -- 

    why not use isRiverIndex here? Can you just do isRiverIndex && riverIndexData?